packages/accounts/src/light-account/account.ts

```diff
@@ -62,16 +65,22 @@ export async function createLightAccount<
 ): Promise<LightAccount<TOwner>>;
 
 export async function createLightAccount({
-  client,
+  transport,
+  chain,
   owner: owner_,
   accountAddress,
   initCode,
   version = "v1.1.0",
-  entrypointAddress = getDefaultEntryPointAddress(client.chain),
-  factoryAddress = getDefaultLightAccountFactoryAddress(client.chain, version),
+  entrypointAddress = getDefaultEntryPointAddress(chain),
+  factoryAddress = getDefaultLightAccountFactoryAddress(chain, version),
   index: index_ = 0n,
 }: CreateLightAccountParams): Promise<LightAccount> {
   let owner = owner_;
+  const client = createBundlerClient({
+    transport,
+    chain,
+  });
```

Comment:
_like_

Reply:
yea I like this a lot more too! good call